application/common/application_data.h

```diff
@@ -158,6 +159,7 @@ class ApplicationData : public base::RefCountedThreadSafe<ApplicationData> {
   bool LoadVersion(base::string16* error);
   bool LoadDescription(base::string16* error);
   bool LoadWindowSetting(base::string16* error);
+  bool LoadDisplayMode(base::string16* error);
```

Comment:
display mode is used only once (at app start), right? what is the reason behind bringing display mode loading from Application?